app/bundles/EmailBundle/Views/Email/form.html.twig

```diff
@@ -0,0 +1,289 @@
+{#
+    @copyright   2014 Mautic Contributors. All rights reserved
+    @author      Mautic
+    @link        http://mautic.org
+    @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+#}
+
+use Symfony\Component\Form\FormView;
+
+{% extends 'MauticCoreBundle:Default:content.html.twig' %}
```

Comment:
Because you are extending another template, everything needs to be contained within a block or Twig doesn't know where to put it.

Reply:
Right